src/compiler/scala/tools/nsc/ast/parser/Parsers.scala

```diff
@@ -2675,7 +2675,7 @@ self =>
         } else {
           accept(EQUALS)
           expr() match {
-            case x if !tp.isEmpty && newmods.isMutable && lhs.toList.forall(_.isInstanceOf[Ident]) && isWildcard(x) =>
+            case x if !tp.isEmpty && newmods.isMutable && lhs.forall(_.isInstanceOf[Ident]) && isWildcard(x) =>
```

Comment:
I noticed this on the doti port, but hadn't noticed your fix-up back here. Thanks!